src/Employer/Employer.Web/Controllers/CreateVacancyOptionsController.cs

```diff
@@ -18,6 +18,16 @@ public CreateVacancyOptionsController(CreateVacancyOptionsOrchestrator orchestra
             _orchestrator = orchestrator;
         }
 
+        [HttpGet("create", Name = RouteNames.CreateVacancyOptions_Create_Get)]
+        public async Task<IActionResult> RedirectToCreateOrOptions([FromRoute]string employerAccountId)
+        {
+            var hasCloneableVacancies = await _orchestrator.HasCloneableVacancies(employerAccountId);
+
```

Comment:
Should this not be performed on the `create-options` action as well?

Reply:
I don't think so. Whenever we want to create a vacancy we just need to point to the `create` endpoint  then this will work out where to redirect to.  the `create-options` endpoint will only ever by called by the redirect in `create`

Reply:
what if `create-options` is typed in.

Reply:
You see the page with an empty list of cloneable vacancies.

Reply:
For good practice I think it should redirect.

Reply:
Personally I don't have a problem with it but will add a redirect 👍